index.html

```diff
@@ -151,5 +151,293 @@ <h2 class="headline-job">I’m a software developer</h2>
         </div>
       </div>
     </header>
+
+    <section class="projects">
+      <h2 class="work-sec-title">My Recent Works</h2>
+      <div class="indicator">
+        <svg
+          width="49"
+          height="4"
+          viewBox="0 0 49 4"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <rect x="0.5" width="48" height="4" rx="2" fill="#36B37E" />
+        </svg>
+      </div>
+      <div class="projects-container">
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
```

Comment:
- [ ] The project title should use ```hx``` tag, this is a requirement. please change the ```<p>``` to a ```<hx>``` tag

Reply:
Noted. I will work on it.